### PR TITLE
[SUPPORT] Allow re-inviting user without assigning to org/space

### PR DESCRIPTION
**Pull request by alext**
## What

When re-inviting a user (eg because they've forgotten their password),
we don't need to specify an org because they'll already be assigned to
the orgs/spaces then need access to. In this case it's desirable to not
touch the orgs/spaces.

This therefore allows this script to run without an org specified when
re-inviting a user.

## How to review

Test that the script works against a dev environment (the `--no-email` flag is useful here) for the following:
* Creating a user assigned to an org
* Re-inviting a user without an org specified

## Who can review

Not me.

### CODE REVIEW

**Comment by alext** [comment]
Closing because this breaks when it comes to email the user. The email template includes the org at present.

**Comment by alext** [comment]
I've addressed the issue with the email template. I've additionally modified the behaviour when `--no-email` is specified so that the email contents are printed, which makes testing easier.

File: scripts/create-user.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ MESSAGE='Hello,
 Your account for GOV.UK PaaS is ready:
 
  - username: ${EMAIL}
- - organisation: ${ORG}
+${ORG:+ - organisation: ${ORG}}
```
Comment (dcarley) [inline scripts/create-user.sh]: Minor: is there a not-terrible way to omit the extra newline that this generates?

Comment (alext) [inline scripts/create-user.sh]: It's possible. I'm not sure about the non-terrible part. Some examples:

```sh
 - username: ${EMAIL}${ORG:+\n - organisation: ${ORG}}
```

```sh
 - username: ${EMAIL}${ORG:+
 - organisation: ${ORG}}
```

What do you think?

Comment (dcarley) [inline scripts/create-user.sh]: The latter seems the best of a bad bunch.

Comment (alext) [inline scripts/create-user.sh]: I've updated it with this version now.

**Comment by dcarley** [comment]
Used and works 👍

**Pull request merged**